app/views/messages/_new_form.html.erb

```diff
@@ -1,4 +1,4 @@
-<%= form_with(model: [message.conversation, message], class: "contents") do |form| %>
+<%= form_with(model: [Current.assistant, message], class: "contents") do |form| %>
```

Comment:
Maybe this `model: [...]` syntax is evidence that we're pushing hard on something because we're off the standard route? Shouldn't rails know how to render this route without this?

Reply:
I think that's okay, I consider that pretty standard rails. It's just the syntax when you have nested routes.